js/main.js

```diff
@@ -0,0 +1,30 @@
+var $avatarURL = document.getElementById('avatar-URL');
+var $avatarImage = document.getElementById('avatarImage');
+
+$avatarURL.addEventListener('input', function (event) {
+  $avatarImage.setAttribute('src', event.target.value);
+});
+
+var $saveButton = document.querySelector('button');
+var $userName = document.getElementById('username');
+var $fullName = document.getElementById('fullName');
+var $location = document.getElementById('location');
+var $bio = document.querySelector('textarea');
+
+$saveButton.addEventListener('click', function (event) {
```

Comment:
You should use the `submit` event on the form element, not the `click` event on the submit button. Don't forget to prevent default!

https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault